internal/server/providers/hostservice/hostservice_e2e_test.go

```
@@ -0,0 +1,376 @@
+// response into queryPayload. Test failures are reported via t.Fatal.
+func postQuery(t *testing.T, base, query string) queryPayload {
+	t.Helper()
+	body, err := json.Marshal(map[string]string{"query": query})
+	if err != nil {
+		t.Fatalf("marshal query: %v", err)
+	}
+	resp, err := http.Post(base+"/graphql", "application/json", strings.NewReader(string(body)))
+	if err != nil {
+		t.Fatalf("POST: %v", err)
+	}
+	defer resp.Body.Close()
```

golangci-lint (2.12.1)

[error] Error return value of `resp.Body.Close` is not checked

(errcheck)